Makefile

```diff
@@ -64,6 +64,9 @@ install-ci:
 	@-bash -c 'pip install -r requirements.txt'
 	@-bash -c 'pip install -e ".[dev]"'
 
+.PHONY: install-all
+install-all: apt install-r-pkg develop
```

Comment:
Do we want to default this behaviour for `all`?

Reply:
I think its fine either way